openapi/pacta.yaml

```diff
@@ -310,6 +310,31 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
 
+  /user/me:
+    get:
+      description: Returns the logged in user, if the user is logged in, otherwise returns empty
+      summary: gets info about the logged in user 
+      operationId: findUserByMe
+      responses:
+        '200':
+          description: user response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        '401':
+          description: caller is not logged in - log in to continue 
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EmptySuccess'
```

Comment:
`EmptySuccess` Is not what I'd call getting a 401, I'd make a new type (or just leave `content` blank!)

Reply:
Fair - previously I couldn't figure out how the return types work with missing `content`! Feel free to change in your upcoming PR to revolutionize errors!

Reply:
I'll leave this to you :)